DiscordBot.DataAccess/EventsSheetsService.cs

```diff
@@ -71,6 +81,31 @@ public async Task WriteRow()
             await request.ExecuteAsync();
         }
 
+        public async Task AddEventAsync(string name, string description, DateTime time)
+        {
+        }
+
+        public async Task EditEventAsync(int key, string? description, string? name, DateTime? time)
```

Comment:
Should the time really be optional? 🙂

Reply:
We wanted to use the method to generalize editing data on the sheet, so the field(s?) that the user updates would be passed as parameters and the rest would be left as is.

Reply:
The idea behind this was to used named parameters and optional arguments, so you could use something like `EditEventAsync(eventKey, name: "NewName)`, and any arguments left as `null` are interpreted as fields that are not being updated. The only required argument would then be `key`, so the service knows what event is being updated.

I think I broke this feature when merging the services together, so I've added it back in d756304 🙂

Reply:
Ah I see! I was thinking of the date as a timestamp for the change, not the date of the event. That makes sense then thanks 🙂